runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperConfig.java

```diff
@@ -232,4 +232,15 @@ default boolean swapTravel()
 	{
 		return true;
 	}
+
+

```

Comment:
Remove the extra newline here--there should be only one empty line between config items.